internal/p2p/peermanager.go

```diff
@@ -39,10 +39,10 @@ const (
 )
 
 // PeerScore is a numeric score assigned to a peer (higher is better).
-type PeerScore uint8
+type PeerScore int
```

Comment:
if we never connect to a node it's score approaches or is 0, but because nothing really increases the score of peers, many nodes that we also connect to often approach 0.

Reply:
Wait a second, do we even set a score when a node is added? Or is it already 0?

Reply:
peer scores default to 0

Reply:
I also don't think we lose anything meaningful by using larger integers? peers could get very big or very small scores over time, maybe.

Reply:
I've made this 16 signed bits as a compromise.

Reply:
Yeah, I don't see any real reason to pinch bits here. 16 bits seems like a fine place to land IMO.